### PR TITLE
Bugfix do not count duplicated domains

### LINKED ISSUE #88: Duplicate entries are counted in Leaderboard
The leaderboard counts duplicate entries. We should at least filter for unique domains tested.

### PR DESCRIPTION
**Pull request by lkappeler**
Fixes #88

### CODE REVIEW

File: js/leaderboard.js [hunk 1]
```diff
@@ -70,22 +70,33 @@
       const existingNicknameEntry = result.find(function(existingEntry) {
         return existingEntry.name === entry.Nickname;
       });
-
+      
       if (existingNicknameEntry) {
+        // only count the same domain once
+        const exsitingDomain = existingNicknameEntry.entries.find((existingEntry) => {
```
Comment (MichaelKohler) [inline js/leaderboard.js]: typo ;)

File: js/leaderboard.js [hunk 2]
```diff
@@ -70,22 +70,33 @@
       const existingNicknameEntry = result.find(function(existingEntry) {
         return existingEntry.name === entry.Nickname;
       });
-
+
```
Comment (MichaelKohler) [inline js/leaderboard.js:73]: please remove

**Review by MichaelKohler — APPROVED** [review]
LGTM; review+ with comments

File: js/leaderboard.js (hunk 1 above)
Comment (lkappeler) [reply js/leaderboard.js]: oh yes indeed, feel free to keep them if you find any more of them ;)

**Review by MichaelKohler — APPROVED** [review]
Thanks!

**Pull request merged**